demos/model_ensemble/python/README.md

```diff
@@ -31,34 +31,34 @@ This guide shows how to implement a model ensemble using the [DAG Scheduler](dag
 ~$ python3 tests/models/argmax_sum.py --input_size 1001 --export_dir ~/models/public/argmax/saved_model
 ```
 
-4. Execute the following commands to convert models to IR format and [prepare models repository](./models_repository.md):
+4. Execute the following commands to convert models to IR format and [prepare models repository](../../../docs/models_repository.md):
 ```
 ~$ docker run -u $(id -u):$(id -g) -v ~/models:/models:rw openvino/ubuntu18_dev:latest deployment_tools/open_model_zoo/tools/downloader/converter.py --name googlenet-v2-tf --download_dir /models --output_dir /models --precisions FP32
 
 ~$ docker run -u $(id -u):$(id -g) -v ~/models:/models:rw openvino/ubuntu18_dev:latest deployment_tools/open_model_zoo/tools/downloader/converter.py --name resnet-50-tf --download_dir /models --output_dir /models --precisions FP32
 
-~$ docker run -u $(id -u):$(id -g) -v ~/models:/models:rw openvino/ubuntu18_dev:latest deployment_tools/model_optimizer/mo_tf.py --input input1,input2 --input_shape [1,1001],[1,1001] --saved_model_dir /models/public/argmax/saved_model/ --output_dir /models/public/argmax/1
+~$ docker run -u $(id -u):$(id -g) -v ~/models:/models:rw openvino/ubuntu18_dev:latest deployment_tools/model_optimizer/mo_tf.py --input input1,input2 --input_shape '[1,1001],[1,1001]' --saved_model_dir /models/public/argmax/saved_model/ --output_dir /models/public/argmax/1
 ~$ mkdir -p ovms_models/googlenet-v2-tf/1
 ~$ mkdir -p ovms_models/resnet-50-tf/1
 ~$ mkdir -p ovms_models/argmax/1
 ~$ cp ~/models/public/googlenet-v2-tf/FP32/googlenet-v2-tf.{bin,xml} ~/ovms_models/googlenet-v2-tf/1/
 ~$ cp ~/models/public/resnet-50-tf/FP32/resnet-50-tf.{bin,xml} ~/ovms_models/resnet-50-tf/1/
 ~$ cp ~/models/public/argmax/1/saved_model.{bin,xml} ~/ovms_models/argmax/1/
 
-~$ tree models/public
+~$ tree ovms_models
```

Comment:
Is it possible to prepare such directory with a Makefile? And also call it `workspace` to match the convention in other demos?

Reply:
done